test/fixtures/commits_after_review.json

```diff
@@ -0,0 +1,56 @@
+{
+  "singleCommit": {
```

Comment:
I would prefer to split this JSON into 4 pieces, `single_commit_after_review_commits.json` & `single_commit_after_review_reviews.json` & `multiple_commits_after_review_commits.json` & `multiple_commits_after_review_commits.json` just like what we do for  the approving reviews/reviewers fitures.